docs/configuration.md

```diff
@@ -54,17 +54,17 @@ Once the data models are accelerated, update the macro definitions next, so that
 
 ![alt]({{ site.baseurl }}/assets/data_models_acceleration.png)
 
-The default definition for the data model macros is summariesonly=**false** and it needs to be changed to summariesonly=**true** (**Settings > Configuration**).
+The default definition for the data model macros is summariesonly=**false**. User can change to summariesonly=**true** if don't want to search non-accelarated data (**Cyences Settings > Cyences App Configuration > Macro Setup > Data-model**).
 
-## **Data Source Macros**
+## **Products Setup (Data Source Macros)**
 
-Navigate to **Settings > Configuration** and underneath the **Data Source Macros** section is where you can view and update several macro definitions. Verify that the macro definitions match the data source (index) used in your Splunk environment.
+Navigate to **Cyences Settings > Cyences App Configuration > Products Setup** where you can view and update several macro definitions. Verify that the macro definitions match the data source (index) used in your Splunk environment.
 
 ![alt]({{ site.baseurl }}/assets/data_source_macros.png)
 
-## **Other Macros**
+## **Macro Setup**
 
-Navigate to **Settings > Configuration** and in the **Other Macros** section is where you can view and update several macro definitions. These macros are used in various alerts, dashboards, searches, and reports. Verify that the macro definitions match the data source (index) used in your environment. 
+Navigate to **Cyences Settings > Cyences App Configuration > Macro Setup** where you can view and update several macro definitions. These macros are used in various alerts, dashboards, searches, and reports. 
 
 | Macro Name | Description | Default Value |            
```

Comment:
Lets add the description of the macro to configuration page itself and then remove these table from doc.